.github/workflows/ci.yml

```diff
@@ -66,6 +66,7 @@ jobs:
           - 12.x
```

Comment:
```suggestion
```

To use the same approach than in IOTAs (tests covering a window of three node versions)